rbac/api/cross_access/view.py

```diff
@@ -127,20 +129,12 @@ def list(self, request, *args, **kwargs):
     def partial_update(self, request, *args, **kwargs):
         """Patch a cross-account request."""
         validate_uuid(kwargs.get("pk"), "cross-account request uuid validation")
-        for field in request.data:
```

Comment:
Are we no longer running this validation on `partial_update`? I see this was moved to `validate_and_format_input`, but that's called from `update`, not `partial_update`? It doesn't look like we used to run this against `update`.

Reply:
Yea we are still having it in the validate_and_format_partial_input, but just for the requestor of the cross access request because the approver could only update the status: 
https://github.com/RedHatInsights/insights-rbac/pull/478/files#diff-9b7f444eec2c52f6eb5578161c2077e0aa9037fbaa6c8ed89eefdcc46dfee381R288